packages/react-component-library/cypress/specs/AutocompleteE/index.spec.ts

```diff
@@ -54,5 +67,22 @@ describe('AutocompleteE', () => {
         options.should('have.length', 0)
       })
     })
+
+    describe('and `z` is typed', () => {
+      beforeEach(() => {
+        cy.get(selectors.selectE.input).type('z')
+      })
+
+      describe('and the tab key is pressed', () => {
+        beforeEach(() => {
+          // @ts-ignore
```

Comment:
Could also add at the top of the file `/// <reference types="cypress-plugin-tab/src" />` (instead of the `// @ts-ignore`) as a workaround for the types not being loaded.